hugo/layouts/index.html

```diff
@@ -16,209 +16,223 @@
+              {{ if (or (not (isset $event_exists_but_not_index $venue.slug)) (not (in (index $event_exists_but_not_index $venue.slug) $year)) ) }}
               {{- with index $venue "volumes_by_year" $year -}}
-                <a href="{{ relref $ (printf "/events/%s-%s.md" $venue.slug $year) }}">{{ slicestr $year 2 4 }}</a>
+              <a href="{{ relref $ (printf " /events/%s-%s.md" $venue.slug $year) }}">{{ slicestr $year 2 4 }}</a>
               {{- end -}}
-            {{ end }}
-            {{ if (and (isset $event_exists_but_not_index $venue.slug) (in (index $event_exists_but_not_index $venue.slug) $year)) }}
+              {{ end }}
+              {{ if (and (isset $event_exists_but_not_index $venue.slug) (in (index $event_exists_but_not_index $venue.slug) $year)) }}
               <span style="color:lightgrey">{{ slicestr $year 2 4 }}</span>
+              {{ end }}
+            </td>
             {{ end }}
-          </td>
+          </tr>
           {{ end }}
-        </tr>
-        {{ end }}
-        {{ end }}
-        {{ range $venue := $.Site.Data.venues }}
-        {{ if (eq $venue.name "Workshops") }}
-        <tr class="text-center border-left border-right">
-          {{ $venue_page := $.Site.GetPage (printf "/venues/%s.md" $venue.slug) }}
-          {{ $acronym := index $venue_page.Params "venue" }}
-          <th scope="row"><a href="{{ $venue_page.RelPermalink }}">{{ $venue.acronym }}</a></th>
-          {{ range $all_years }}
-          {{ $year := string . }}
-          {{/* this is the actual default case */}}
-          <td class="{{ cond (in $border_years $year) "border-left" "" }} {{ cond (eq "1971" $year) "border-right" "" }}">
-            {{ if (or (not (isset $event_exists_but_not_index $venue.slug)) (not (in (index $event_exists_but_not_index $venue.slug) $year)) ) }}
+          {{ end }}
+          {{ range $venue := $.Site.Data.venues }}
+          {{ if (eq $venue.name "Workshops") }}
+          <tr class="text-center border-left border-right">
+            {{ $venue_page := $.Site.GetPage (printf "/venues/%s.md" $venue.slug) }}
+            {{ $acronym := index $venue_page.Params "venue" }}
+            <th scope="row"><a href="{{ $venue_page.RelPermalink }}">{{ $venue.acronym }}</a></th>
+            {{ range $all_years }}
+            {{ $year := string . }}
+            {{/* this is the actual default case */}}
+            <td class="{{ cond (in $border_years $year) " border-left" "" }} {{ cond (eq "1971" $year) "border-right" "" }}">
+              {{ if (or (not (isset $event_exists_but_not_index $venue.slug)) (not (in (index $event_exists_but_not_index $venue.slug) $year)) ) }}
               {{- with index $venue "volumes_by_year" $year -}}
-                <a href="{{ relref $ (printf "/events/%s-%s.md" $venue.slug $year) }}">{{ slicestr $year 2 4 }}</a>
+              <a href="{{ relref $ (printf " /events/%s-%s.md" $venue.slug $year) }}">{{ slicestr $year 2 4 }}</a>
```

Comment:
Is that space intended?

Reply:
no, this was broken auto formatting by my editor (fixed in new commit)